lib/Sema/TypeCheckDecl.cpp

```diff
@@ -676,10 +676,16 @@ IsStaticRequest::evaluate(Evaluator &evaluator, FuncDecl *decl) const {
       dc->isTypeContext()) {
     const auto operatorName = decl->getBaseIdentifier();
     if (auto ED = dyn_cast<ExtensionDecl>(dc->getAsDecl())) {
-      decl->diagnose(diag::nonstatic_operator_in_extension,
-                     operatorName, ED->getExtendedTypeRepr())
-          .fixItInsert(decl->getAttributeInsertionLoc(/*forModifier=*/true),
-                       "static ");
+        if (ED->getExtendedTypeRepr()) {
+            decl->diagnose(diag::nonstatic_operator_in_extension,
+                           operatorName, ED->getExtendedTypeRepr())
+                .fixItInsert(decl->getAttributeInsertionLoc(/*forModifier=*/true),
+                             "static ");
+        } else {
+            decl->diagnose(diag::nonstatic_operator_in_extension_no_type, operatorName)
+                .fixItInsert(decl->getAttributeInsertionLoc(/*forModifier=*/true),
+                             "static ");
+        }
```

Comment:
The convention is 2 spaces for the indent width (not 4).

Reply:
Fixed

Reply:
Could you also run clang-format on your changes to make sure everything is properly formatted and indented?

Reply:
Will do